waflib/Tools/qt5.py

```diff
@@ -395,13 +395,37 @@ def scan(self):
 				names.append(x)
 		return (nodes, names)
 
+	def quote_flag(self, x):
+		"""
+		QT parses the argument file line-by-line, so no need to escape or quote
```

Comment:
I'd change docstring to say something about how we override the Task() class quote_flag since the behavior of qt moc/rcc are different from that of native ms tools like cl.exe/link.exe